### PR TITLE
Collapsible menu

### PR DESCRIPTION
**Pull request by SebasDuranona**
Added styles, and basic functionality for the collapsible navBar.
Added FontAwesome for the icon.

### CODE REVIEW

**Review by jhuppe — CHANGES_REQUESTED** [review]
Visually it looks great. I really like the use of the diamond font awesome icon. If you open up the console when you have the project running you will notice you have a few console errors. 

It looks like you are nesting <a> tags incorrectly. 
`Warning: validateDOMNesting(...): <a> cannot appear as a descendant of <a>.
    in a (at NavBar.js:82)
    in a (created by Context.Consumer)
    in Link (at NavBar.js:82)
    in div (at NavBar.js:81)
    in div (at NavBar.js:80)
    in li (at NavBar.js:77)
    in ul (at NavBar.js:69)
    in div (at NavBar.js:68)
    in div (at NavBar.js:7)
    in NavBar (at App.js:23)
    in div (at App.js:22)
    in App (at src/index.js:10)
    in Router (created by BrowserRouter)
    in BrowserRouter (at src/index.js:9)`

**Comment by SebasDuranona** [comment]
Yeah I see it...
Looks like a problem related to the Link tags
I'll check it out right away

**Comment by SebasDuranona** [comment]
Okay, my last commit should've dealt with the issue

File: src/Components/NavBar.js
Comment (jhuppe) [inline src/Components/NavBar.js]:
```suggestion
              <label id='collapsible-btn' htmlFor="menu"><FontAwesomeIcon icon="gem" /></label>
```
This is the last error that is showing in the console. You need to use htmlFor instead of for.

Comment (SebasDuranona) [reply src/Components/NavBar.js]: Done 😄

File: src/App.js
Comment (jhuppe) [inline src/App.js]:
```suggestion

```
You don't ever use this variable, so you should just remove that line.

**Comment by jhuppe** [comment]
Looks good to merge once the merge conflict is resolved.

**Review by jhuppe — APPROVED** [review]

**Pull request merged**
